### ISSUE TITLE
Does not work on Ubuntu 23.10.1

### ISSUE THREAD

**Issue by MechMK1** [marcinjahn/gnome-peek-top-bar-on-fullscreen-extension#14]
I've installed the extension and ensured it is enabled. However, when going fullscreen on various applications, the extension does not "activate".

I don't know where relevant log files are located that could help with this issue. I'll add them as directed.

**Comment by MechMK1** [comment]
The output of the command is

```
dummy-window.js  extension.js  metadata.json
```

The output of `journalctl` is also uninteresting:

```
Jän 11 12:39:32 hostname gnome-shell[2213]: Disabling extension peek-top-bar-on-fullscreen@marcinjahn.com
Jän 11 12:39:37 hostname systemd[1]: NetworkManager-dispatcher.service: Deactivated successfully.
Jän 11 12:39:37 hostname gnome-shell[2213]: Enabling extension peek-top-bar-on-fullscreen@marcinjahn.com
```

**Comment by marcinjahn** [comment]
Yeah, nothing suspicious there. What apps do you actually try to use in full screen? Chrome works alright for me, try that and see if anything changes. If you have Chrome installed, open it and press <kbd>F11</kbd> to go to full screen. Top panel should show up when you hover your mouse to the top

**Comment by MechMK1** [comment]
I tried it with Chrome, to no avail. I also tried it with Binding of Isaac and other games off Steam, which also didn't work.

I wish I could be more helpful, but I don't see any log entries that give any indication what could be wrong. It just doesn't trigger.

**Comment by marcinjahn** [comment]
Interesting. I'm not sure games would work, I didn't test that use case.
Are you using X or Wayland?

**Comment by MechMK1** [comment]
An alternative extension [Fullscreen Avoider](https://github.com/Noobsai/fullscreen-avoider) does work with all games I've tested thus far, so I *believe* that games should work just fine.

I'm currently using X11, as is the default for Ubuntu 23.10.1

Note: I've uninstalled Fullscreen Avoider prior to all tests for this extension, just in case you were worried about conflicts

**Comment by marcinjahn** [comment]
Alright, I have to admit I haven't tested the extension on X since some time, I'll have a look.

**Comment by MechMK1** [comment]
Sounds like X may be the source of the trouble then. I was halfway expecting some form of unhandled event being the culprit in this case.

**Comment by marcinjahn** [comment]
Could be, although X has an explicit support added [here](https://github.com/marcinjahn/gnome-peek-top-bar-on-fullscreen-extension/blob/main/src/panel/x11-panel-manager.ts). Actually, making this extension work was much easier on X11 than on Wayland :slightly_smiling_face:, but potentially later on something got broken there.

**Comment by marcinjahn** [comment]
I checked on X11, it does work. It even works better than on Wayland, panel shows up on top of Firefox, whereas on Wayland that doesn't happen for some reason.
Unfortunately I do not have any Ubuntu installation to test it out.
